Felienne/hedy
# [BUG FIX] Correctly hides stop button on timeout

**Description**
In this PR we fix an issue where the stop button wouldn't be hidden on a timeout, making the user unable to run another program.

**Fixes**
This PR fixes #2612.

**How to test**
Make sure you cause a "natural" timeout in some way and verify that the stop button is correctly hidden and the run button is shown again. One easy way to achieve this is by running the following program in level 7:

`repeat 500 times print 'Hedy is fun!'`
